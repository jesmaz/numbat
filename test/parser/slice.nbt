[:]
[a:]
[a:b]
[a:b:c]
[a:b] & [c:d]
[a:b] | [c:d]
[a:b] ^ [c:d]
[a in b]
a [:]
a [:] = b [c:d]
a [:] = b [c:d] | e [f:g]
a [:] = b
a [1:50:2]