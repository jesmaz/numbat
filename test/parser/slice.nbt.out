[:]
[a :]
[a : b]
[a : b : c]
([a : b] & [c : d])
([a : b] | [c : d])
([a : b] ^ [c : d])
[a in b]
(a [:])
((a [:]) = (b [c : d]))
((a [:]) = ((b [c : d]) | (e [f : g])))
((a [:]) = b)